### PR TITLE
[Merged by Bors] - chore: revert leantar #5710 for further testing

### PR DESCRIPTION
**Pull request by kim-em**
This reverts commit 6ed1d668fb2c694b26798fb65c66f60017e8b47a.

Currently broken on gitpod (needs `sudo apt-get install xz-utils`) and on Windows (likely the same issue).

---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: Cache/IO.lean [hunk 1]
```diff
@@ -155,44 +136,6 @@ def validateCurl : IO Bool := do
...
-      if System.Platform.getIsOSX () then s!"{arch}-apple-darwin"
-      else s!"{arch}-unknown-linux-gnu"
-  IO.println s!"leantar is too old; downloading more recent version"
-  IO.FS.createDirAll IO.CACHEDIR
-  let ext := if win then "zip" else "tar.xz"
-  let _ ← runCmd "curl" #[
-    s!"https://github.com/digama0/leangz/releases/download/v{LEANTARVERSION}/leantar-v{LEANTARVERSION}-{target}.{ext}",
-    "-L", "-o", s!"{LEANTARBIN}.{ext}"]
-  let _ ← runCmd "tar" #["-xf", s!"{LEANTARBIN}.{ext}",
-    "-C", IO.CACHEDIR.toString, "--strip-components=1"]
```
Comment (eric-wieser) [inline Cache/IO.lean:193]: Does switching to use gzip instead of xz here suffice?

Comment (eric-wieser) [reply Cache/IO.lean:193]: Ah, I guess the problem is that @digama0 used xz for the releases here: https://github.com/digama0/leangz/releases/tag/v0.1.1

**Comment by jcommelin** [comment]
Thanks :tada: 

If CI passes, please remove the label `awaiting-CI` and merge this yourself, by adding a comment `bors r+`.

bors d+

**Comment by semorrison** [comment]
bors merge

**Pull request closed without merging**